# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1204794244 18000
#      Thu Mar 06 04:04:04 2008 -0500
# Branch release-3-0-x
# Node ID e6364723ae29e7430b078f1eb9a092881bbfd2f6
# Parent  a5a86cc9ef38ad97dfc2201e90c4c2fad68ee132
markersize compatibility changes

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-03-06  John W. Eaton  <jwe@octave.org>
+
+	* plot/__go_draw_axes__.m: Adjust markersize by a factor of 1/6.
+
 2008-03-05  Ben Abbott  <bpabbott@mac.com>
 
 	* polynomial/roots.m: Catch Infs and/or NaNs.
 
 2008-03-05  Sebastien Loisel  <loisel@temple.edu>
 
 	* polynomial/roots.m: Apply a scaling factor to the removal of the
 	leading zeros.
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -635,17 +635,17 @@ function __go_draw_axes__ (h, plot_strea
 	     data_idx++;
              is_image_data(data_idx) = false;
              parametric(data_idx) = false;
 	     have_cdata(data_idx) = false;
              titlespec{data_idx} = "title \"\"";
 	     usingclause{data_idx} = "";
 
 	     if (isfield (obj, "markersize"))
-	       mdat = obj.markersize;
+	       mdat = obj.markersize / 6;
 	     endif
 
              if (isfield (obj, "edgecolor"))
 	       if ((strncmp (obj.edgecolor, "flat", 4)
 		    || strncmp (obj.edgecolor, "interp", 6))
 		   && isfield (obj, "cdata"))
 		 if (ndims (obj.cdata) == 2
 		     && (size (obj.cdata, 2) == nc
@@ -750,19 +750,19 @@ function __go_draw_axes__ (h, plot_strea
 		 style = "points";
 	       endif
 	     elseif (! isempty (pt))
 	       style = "linespoints";
 	     endif
 
 	     if (isfield (obj, "markersize"))
 	       if (length (mdat) == nc)
-		 m = mdat(i);
+		 m = mdat(i) / 6;
 	       else
-		 m = mdat;
+		 m = mdat / 6;
 	       endif
 	       if (! strcmpi (style, "lines"))
 		 if (have_newer_gnuplot)
 		   ps = sprintf("pointsize %f", m);
 		 else
 		   ps = sprintf("ps %f", m);
 		 endif
 	       else
@@ -1404,20 +1404,20 @@ function [style, typ, with] = do_linesty
 	style = "points";
       endif
     elseif (! isempty (pt))
       style = "linespoints";
     endif
 
     if (isfield (obj, "markersize"))
       if (have_newer_gnuplot)
-	fprintf (plot_stream, " pointsize %f", obj.markersize);
+	fprintf (plot_stream, " pointsize %f", obj.markersize /6);
       else
 	if (! strcmpi (style, "lines"))
-	  with = sprintf ("%s ps %f", with, obj.markersize);
+	  with = sprintf ("%s ps %f", with, obj.markersize / 6);
 	endif
       endif
       found_style = true;
     endif
   else
     style = errbars;
     found_style = true;
   endif
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,17 @@
+2008-03-06  John W. Eaton  <jwe@octave.org>
+
+	* graphics.cc (line::properties::properties,
+	line::properties::factory:defaults, patch::properties::properties, 
+	patch::properties::factory:defaults,
+	surface::properties::properties,
+	surface::properties::factory:defaults):
+	Set markersize to 6 by default.
+
 2008-03-06  Alexander Barth  <barth.alexander@gmail.com>
 
 	* DLD-FUNCTIONS/__lin_interpn__.cc (lookup):
 	Handle decreasing coordinate values.
 
 2008-02-27  John W. Eaton  <jwe@octave.org>
 
 	* oct-stream.cc (do_read): Stop reading if seek fails.
